docs/_future/manifest_functions.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Manifest functions
```

Comment:
I'm not super familiar with the implementation on the Python side of this, but can Functions only be initialized in the Manifest? Does it make sense to call them "Manifest Functions" or just "Functions"? Wondering if it might cause confusion with the slightly different terminology between here, the API site, and the Bolt JS site!

Reply:
Because the Manifest is defined in the `manifest.json` there is somewhat of a line between the `python code function` and the `manifest json function`. I was trying to refer to the `python code function` as the "Function" and the one in the manifest as the "Function Definition". @hello-ashleyintech I'm not sure if this is the best way to look at it.

Reply:
ahhh yes, I see! that makes sense to me. I'll take a look over this again either later today or tomorrow and let you know if I have any different thoughts about the wording or how to make that more clear for readers, but currently it sounds good to me! 👍